### PR TITLE
Check that file operation is possible within account space limit

### PR DESCRIPTION
**Pull request by kevodwyer**

### CODE REVIEW

**Comment by ianopolous** [comment]
It looks like both errors say, "File copy operation exceeds Account quota" Should one be talking about upload?

Also I don't think we should use the term quota in the UI (I don't think we currently use it anywhere?). Maybe something like
"File copy operation exceeds total space. Please upgrade to get more space."

**Comment by kevodwyer** [comment]
> It looks like both errors say, "File copy operation exceeds Account quota" Should one be talking about upload?
> 
> Also I don't think we should use the term quota in the UI (I don't think we currently use it anywhere?). Maybe something like "File copy operation exceeds total space. Please upgrade to get more space."

done

File: src/views/Drive.vue [hunk 1]
```diff
@@ -1038,38 +1038,43 @@ module.exports = {
...
-                        that.$toast({component: ProgressBar,props:  progress} , { icon: false , timeout:false, id: files[i].name})
-                        progressBars.push(progress);
-                    }
-                    const uploadParams = {
-                        applyReplaceToAll: false,
-                        replaceFile: false,
-                        filesUploaded: 0
+                    let spaceAfterOperation = this.checkAvailableSpace(totalSize);
+                    if (spaceAfterOperation < 0) {
+                        let errMsg = "File copy operation exceeds available space\n" + "Please free up " + helpers.convertBytesToHumanReadable('' + -spaceAfterOperation) + " and try again";
```
Comment (ianopolous) [inline src/views/Drive.vue:1047]: is this a file upload or copy?

Comment (kevodwyer) [reply src/views/Drive.vue]: upload. have now fixed message

**Review by ianopolous — APPROVED** [review]

**Pull request merged**